yt/yt/server/job_proxy/unittests/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ target_link_options(unittester-job-proxy PRIVATE
 
 target_sources(unittester-job-proxy PRIVATE
   ${PROJECT_SOURCE_DIR}/yt/yt/server/job_proxy/unittests/asan_warning_filter_ut.cpp
+  ${PROJECT_SOURCE_DIR}/yt/yt/server/job_proxy/unittests/stderr_writer.cpp
```

Comment:
stderr_writer_ut.cpp

Did you check compilation of this code after PR updates?